# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
NaroBody_1_NaroBody_2_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: NaroBody_1_NaroBody_2_joint
  pid: {p: 10.0, i: 1.0, d: 0.0}
NaroBody_2_NaroBody_3_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: NaroBody_2_NaroBody_3_joint
  pid: {p: 10.0, i: 1.0, d: 0.0}
NaroBody_3_Naro_caudal_fin_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: NaroBody_3_Naro_caudal_fin_joint
  pid: {p: 10.0, i: 1.0, d: 0.0}
NaroBody_1_Naro_pectoral_fin_right_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: NaroBody_1_Naro_pectoral_fin_right_joint
  pid: {p: 1.0, i: 1.0, d: 0.0}
NaroBody_1_Naro_pectoral_fin_left_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: NaroBody_1_Naro_pectoral_fin_left_joint
  pid: {p: 1.0, i: 1.0, d: 0.0}



